### ISSUE TITLE
Credentials life cycle management - incl. automated renewal, etc.

### ISSUE THREAD

**Issue by dumblob** [w3c-ccg/community#232]
I could not find a better place to ask. I am new to W3C CCG and would like to understand the scope of the WG.

Namely I am interested in the **complete** lifecycle of credentials of a given person. We stumbled upon this issue in several corners of the internet - see some chosen links below.

https://github.com/w3c/webappsec-change-password-url/issues/34
https://github.com/dani-garcia/vaultwarden/discussions/1691
https://community.bitwarden.com/t/one-click-password-changer/14619/7
https://w3c.github.io/webappsec-change-password-url/

Is the **complete** management of credentials fully in scope of this WG. If so, where can I find more information? If not, could you point me to a relevant WG?

Thanks!

**Comment by wip-abramson** [comment]
Hey, 
It seems this issue slipped though, wondering if you managed to find what you were looking for elsewhere @dumblob?

Also, I think these sorts of questions are best asked on the CCG mailing list - public-credentials@w3.org in future

**Comment by dumblob** [comment]
Hi @wip-abramson , I did not find any answer to this.

Searching in public-credentials@w3.org public mail archive (https://lists.w3.org/Archives/Public/public-credentials/ ), I failed to find anything related. Are you participating in the regular WG meetings?

**Comment by msporny** [comment]
> Is the complete management of credentials fully in scope

Yes, it is -- the primary spec we work on for full credential lifecycle management is the VC API:

https://w3c-ccg.github.io/vc-api/

We do cover credential refresh here:

https://w3c-ccg.github.io/vc-refresh-2021/
